File: 1894-merge-strings-alternately/merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int n = min(word1.size(), word2.size());
        int max_n = max(word1.size(), word2.size());
        bool Big1 = (word1.size() > word2.size());
        string ans = "";
        for(int i = 0; i < n; i ++)
        {
            ans += word1[i];
            ans += word2[i];
        }
        if(Big1)
        {
            ans += word1.substr(n, max_n-n);
        }
        else
        {
            ans += word2.substr(n, max_n-n);
        }
        return ans;
    }
};

File: 1894-merge-strings-alternately/merge-strings-alternately.py
class Solution(object):
    def mergeAlternately(self, word1, word2):
        """
        :type word1: str
        :type word2: str
        :rtype: str
        """
        ans = ""
        i,j= 0,0
        for _ in range(len(word1 + word2)):
            if i < len(word1):
                ans += word1[i]
                i+=1
            if j < len(word2):
                ans += word2[j]
                j+=1
        return ans


        